preliz/distributions/gamma.py

```diff
@@ -0,0 +1,211 @@
+            self.params = (self.mu, self.sigma)
+
+        self.is_frozen = True
+
+    def _from_mu_sigma(self, mu, sigma):
+        alpha = mu**2 / sigma**2
+        beta = mu / sigma**2
+        return alpha, beta
+
+    def _to_mu_sigma(self, alpha, beta):
+        mu = alpha / beta
+        sigma = alpha**0.5 / beta
+        return mu, sigma
+
+    def pdf(self, x):
+        """
+        Compute the probability density function (PDF) at a given point x.
+        """
+        x = np.asarray(x)
+        return np.exp(nb_logpdf(x, self.alpha, self.beta))
+
+    def cdf(self, x):
+        """
+        Compute the cumulative distribution function (CDF) at a given point x.
+        """
+        x = np.asarray(x)
+        return nb_cdf(x, self.alpha, self.beta, 0, np.inf)
+
+    def ppf(self, q):
+        """
+        Compute the percent point function (PPF) at a given probability q.
+        """
+        q = np.asarray(q)
+        return nb_ppf(q, self.alpha, self.beta, 0, np.inf)
+
+    def logpdf(self, x):
+        """
+        Compute the log probability density function (log PDF) at a given point x.
+        """
+        return nb_logpdf(x, self.alpha, self.beta)
+
+    def _neg_logpdf(self, x):
+        """
+        Compute the neg log_pdf sum for the array x.
+        """
+        return nb_neg_logpdf(x, self.alpha, self.beta)
+
+    def entropy(self):
+        return nb_entropy(self.alpha, self.beta)
+
+    def mean(self):
+        return self.alpha / self.beta
+
+    def median(self):
+        return self.ppf(0.5)
+
+    def var(self):
+        return self.alpha / self.beta**2
+
+    def std(self):
+        return self.var() ** 0.5
+
+    def skewness(self):
+        return 2 / self.alpha**0.5
+
+    def kurtosis(self):
+        return 6 / self.alpha
+
+    def rvs(self, size=None, random_state=None):
+        random_state = np.random.default_rng(random_state)
+        return random_state.gamma(self.alpha, 1 / self.beta, size)
+
+    def _fit_moments(self, mean, sigma):
+        alpha, beta = self._from_mu_sigma(mean, sigma)
+        self._update(alpha, beta)
+
+    def _fit_mle(self, sample, **kwargs):
+        optimize_ml(self, sample)
+
+
+def nb_cdf(x, alpha, beta, lower, upper):
+    prob = gammainc(alpha, x / (1 / beta))
+    return cdf_bounds(prob, x, lower, upper)
+
+
+def nb_ppf(q, alpha, beta, lower, upper):
+    x_val = gammaincinv(alpha, q) * (1 / beta)
+    return ppf_bounds_cont(x_val, q, lower, upper)
+
+
+@nb.njit(cache=True)
+def nb_logpdf(x, alpha, beta):
+    x = x / (1 / beta)
+    return xlogy(alpha - 1.0, x) - x - gammaln(alpha) - np.log(1 / beta)
+
+
+@nb.njit(cache=True)
+def nb_neg_logpdf(x, alpha, beta):
+    return -(nb_logpdf(x, alpha, beta)).sum()
+
+
+@nb.njit(cache=True)
+def nb_entropy(alpha, beta):
+    return (
+        0.5 * (1.0 + np.log(2 * np.pi) + np.log(alpha))
+        - 1 / (3 * alpha)
+        - (alpha**-2.0) / 12
+        - (alpha**-3) / 90
+        + (alpha**-4) / 120
+    ) + np.log(1 / beta)
```

Comment:
```suggestion
    return alpha - np.log(beta) + gammaln(alpha) + (1 - alpha) * digamma(alpha)
```

Import digamma from preliz.internal.special